package.json

```diff
@@ -1,13 +1,15 @@
 {
   "private": true,
   "scripts": {
+    "prefresh": "npm run clean || echo 'clean failed; no worries!'",
```

Comment:
This message is a little confusing to me - if the clean fails but it doesn't matter either way should we be messaging them at all? Is there anything they need to do if the clean fails?